.github/workflows/nm-build-test.yml

```diff
@@ -129,7 +129,7 @@ jobs:
             timeout: ${{ inputs.test_timeout }}
             gitref: ${{ github.ref }}
             python: ${{ matrix.test_config.python }}
-            whl: ${{ needs.BUILD.output.whl }}
+            whl: ${{ needs.BUILD.outputs.whl }}
```

Comment:
we can likely delete this parameter all together, since we are finding the whl without using the name.

Reply:
We need to download the wheel from GH artifactory and install it during test so still need it here in order to know the wheel name.